gcp/modules/gpii-flushtokens/values.yaml

```diff
@@ -0,0 +1,7 @@
+image:
+  repository: ${flushtokens_repository}
+  checksum: ${flushtokens_checksum}
+
+couchdbUrl: "http://${couchdb_admin_username}:${couchdb_admin_password}@couchdb-svc-couchdb.gpii.svc.cluster.local:5984/gpii"
+
+cronJobSchedule: "*/15 * * * *"
```

Comment:
Can you please move this default to ` shared/charts/gpii-flushtokens/values.yaml`?

Reply:
Done.